### DISCUSSION TITLE
list of compatible devices

### DISCUSSION THREAD

**Discussion by eadmaster** [jasonacox/tinytuya#487] [Q&A]
since there are many variations of these tuya products, having a list of compatible devices with some way to identify them/purchase link would help.

I am especially interested in [these combined IR+RF bridges](https://it.aliexpress.com/item/1005005940538953.html?utparam-url=scene%3Asearch%7Cquery_from%3A).

**Comment by uzlonewolf** [comment]
Unfortunately it's just not feasible to do this.  Devices change all the time and even if you buy a 2nd one from the exact same listing the one you receive could be completely different than the original one.

That particular device should be covered by Contrib/RFRemoteControlDevice https://github.com/jasonacox/tinytuya/blob/master/examples/Contrib/RFRemoteControlDevice-example.py

**Reply by eadmaster** [reply]
i see, sometimes these devices have some associated code like "S11" in the listing i've linked.
Even if this won't garantee compatibility i think it may still help picking up the right product.

Note that a compatibility list could also include **negative** entries: e.g.  devices that do **not** work, which again could help saving some time picking the right product.
